xarray/backends/pydap_.py

```diff
@@ -131,7 +131,11 @@ def open(
 
     def open_store_variable(self, var):
         data = indexing.LazilyIndexedArray(PydapArrayWrapper(var))
-        return Variable(var.dimensions, data, _fix_attributes(var.attributes))
+        return Variable(
```

Comment:
@Mikejmnez does this look right to you?

Reply:
Hey @Mikejmnez, this change was working for my use-case, but it looks like the real fix is ever-so-slightly more complicated than my patch here ;). Closing.

Reply:
No problems, and thanks a lot @Zeitsperre for the attempted fix :) .  Indeed that would have gotten rid of the slashes on dim names +1